web/src/app/modules/views/main/editors/modules/contents-container/components/ceg-model-container.component.ts

```diff
@@ -57,11 +59,22 @@ export class CEGModelContainer extends TestSpecificationContentContainerBase<CEG
                 await this.dataService.deleteElement(element.url, true, Id.uuid);
                 await this.dataService.commit(this.translate.instant('save'));
                 await this.dataService.deleteCachedContent(element.url);
-                await this.modal.openOk(this.translate.instant('CEGGenerator.couldNotGenerateTitle'), 
+                await this.modal.openOk(this.translate.instant('CEGGenerator.couldNotGenerateTitle'),
                         this.translate.instant('CEGGenerator.couldNotGenerate'));
                 return undefined;
             }
         }
         return element;
     }
+

```

Comment:
This is a clone to the process model container. Both containers have a common base class. Thus, the method and the reference to the ContentService should be moved up.